phoenix_static_buildpack.config

```diff
@@ -0,0 +1,2 @@
+# Node version
+node_version=10.15.3
```

Comment:
Not big issue but might be good to bump to a more recent version > 12.x.x

Reply:
Updated to `12.13.0`